beetsplug/export.py

```diff
@@ -78,17 +100,22 @@ def commands(self):
             u'-o', u'--output',
             help=u'path for the output file. If not given, will print the data'
         )
+        cmd.parser.add_option(
+            u'-f', u'--format', default='json',
+            help=u"the output format: json (default), csv, or xml"
+        )
         return [cmd]
 
     def run(self, lib, opts, args):
-
         file_path = opts.output
-        file_format = self.config['default_format'].get(str)
         file_mode = 'a' if opts.append else 'w'
+        file_format = opts.format if opts.format else \
+            self.config['default_format'].get(str)
```

Comment:
This should suffice: `opts.format or self.config['default_format'].get(str)`

Reply:
sounds good